contrib/mod_sftp/fxp.c

```diff
@@ -1097,10 +1097,10 @@ static const char *fxp_get_request_type_desc(unsigned char request_type) {
       return "CLOSE";
 
     case SFTP_SSH2_FXP_READ:
-      return "READ";
+      return "READFILE";
```

Comment:
This will change the names as logged, which in turn will have consequences for any log parsers/regexes that folks may be relying upon.